Makefile

```
@@ -32,7 +32,6 @@ bin:
 	@mkdir -p $@
 
 .PHONY: driver
-bin/ibm-powervs-block-csi-driver:
 driver: | bin
 	CGO_ENABLED=0 GOOS=linux GOARCH=$(ARCH) go build -ldflags ${LDFLAGS} -o bin/ibm-powervs-block-csi-driver ./cmd/
```

both GOOS and GOARCH can be removed which is won't make much difference anymore after buildx implementation.

```suggestion
	CGO_ENABLED=0 go build -ldflags ${LDFLAGS} -o bin/ibm-powervs-block-csi-driver ./cmd/
```

with this you need to remove TARGETPLATFORM  and ARCH from here: https://github.com/kubernetes-sigs/ibm-powervs-block-csi-driver/blob/fe391d360ee8b3158bca0c92de40e4ed3992572a/Dockerfile#L18-L19

---

fixed